test/model/src/driver/kv_workload.cpp

```diff
@@ -172,6 +172,10 @@ parse(const char *str)
         return insert(parse_uint64(args[0]), parse_uint64(args[1]),
           data_value(parse_uint64(args[2])), data_value(parse_uint64(args[3])));
     }
+    if (name == "get") {
```

Comment:
I noticed that this list should be in alphabetic order - so "get" op should go before "insert"

Reply:
When I first looked at it, it wasn’t obvious to me that 'get' meant the same as 'search', curious why 'get' was chosen over 'search', since 'search' matches WT terminology better.

Reply:
I was copying the C++ model's table method name. In this folder, all the helpers that invoke search are called "get".

Also sure, I didn't notice they were in alphabetical order at all - I was grouping by theme instead. Don't worry about pointing out all 3+ other places this comes up, I'll change it consistently.